subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+
+    def postprocess(self):
+        # Inspect container for --tmpfs
+        subargs = ["--format", "'{{.HostConfig.Tmpfs}}'",
```

Comment:
Not sure you need the double and single-quotes here, though python may complain about the curly-braces (escape them with ``\`` if so).  Argument-passing in a list works differently in this respect than a shell.  In act, the list-format is the same in C/C++ and other languages as well.